comps/asr/src/integrations/opea_whisper_asr.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Based on our naming rules, this name should be `opea_whisper.py`, and the test scripts name should be `tests/asr/test_asr_opea_whisper.sh`

Reply:
Thanks, done.